src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -0,0 +1,1766 @@
+                //             case not [{} x] { {} y, .. {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "x").WithArguments("x").WithLocation(8, 53),
+                // (8,56): error CS0165: Use of unassigned local variable 'y'
+                //             case not [{} x] { {} y, .. {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "y").WithArguments("y").WithLocation(8, 56),
+                // (8,59): error CS0165: Use of unassigned local variable 'z'
+                //             case not [{} x] { {} y, .. {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "z").WithArguments("z").WithLocation(8, 59),
+                // (9,18): error CS8120: The switch case is unreachable. It has already been handled by a previous case or it is impossible to match.
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_SwitchCaseSubsumed, "[not {} x] { not {} y, .. not {} z }").WithLocation(9, 18),
+                // (9,26): error CS8780: A variable may not be declared within a 'not' or 'or' pattern.
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_DesignatorBeneathPatternCombinator, "x").WithLocation(9, 26),
+                // (9,38): error CS8780: A variable may not be declared within a 'not' or 'or' pattern.
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_DesignatorBeneathPatternCombinator, "y").WithLocation(9, 38),
+                // (9,51): error CS8780: A variable may not be declared within a 'not' or 'or' pattern.
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_DesignatorBeneathPatternCombinator, "z").WithLocation(9, 51),
+                // (9,61): error CS0165: Use of unassigned local variable 'x'
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "x").WithArguments("x").WithLocation(9, 61),
+                // (9,64): error CS0165: Use of unassigned local variable 'y'
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "y").WithArguments("y").WithLocation(9, 64),
+                // (9,67): error CS0165: Use of unassigned local variable 'z'
+                //             case [not {} x] { not {} y, .. not {} z }: _ = (x, y, z); break;
+                Diagnostic(ErrorCode.ERR_UseDefViolation, "z").WithArguments("z").WithLocation(9, 67));
+        }
+
+        [Fact]
+        public void ListPattern_Negated_02()
```

Comment:
:memo: definitely=true has no effect in this test because of special handling of `is`. That's covered by `ListPattern_Negated_01`